daml-assistant/scala-daml-project-config/src/main/scala/com/digitalasset/assistant/config/ProjectConfig.scala

```diff
@@ -7,9 +7,13 @@ import java.io.File
 import java.nio.file.{Files, Path}
 
 import io.circe.{Json, yaml}
+import io.circe.optics.JsonOptics._
+import monocle.function.Plated
```

Comment:
:eyes:

Reply:
Yeah turns out this is recommended on the circe docs :D